storeshare_api/config/application.rb

```diff
@@ -11,6 +13,10 @@ class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
     config.load_defaults 7.0
 
+    # config.session_store :cookie_store, key: '_interslice_session'
+    # config.middleware.use ActionDispatch::Cookies
+    # config.middleware.use config.session_store, config.session_options
```

Comment:
it's possible we'll still need to use session store cookies vs jwt tokens but i'll take care of figuring that out

Reply:
replaced with JWT